gensim/models/wrappers/wordrank.py

```diff
@@ -118,14 +117,14 @@ def train(cls, wr_path, corpus_file, out_name, size=100, window=15, symmetric=1,
                     utils.check_output(w, args=command, stdin=r)
 
         logger.info("Deleting frequencies from vocab file")
-        with smart_open(vocab_file, 'wb') as w:
+        with smart_open(join(meta_dir, vocab_file), 'wb') as w:
```

Comment:
plese move `join` to defenition of `vocab_file` (line 91) and same changes for all `smart_open` arguments